core/evancore.vue.libs/src/components/forms/checkbox/checkbox.vue

```diff
@@ -38,7 +38,7 @@
         type="checkbox"
         v-bind="$attrs"
       >
-      <label :for="id" />
+      <label />
```

Comment:
Is this really sufficient? Will a click on the label focus the corresponding input?

Reply:
The label at this point is just for styling the "fancy" checkbox. When the id property was specified, the design does not change anymore, so it's not needed for the label.